docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -451,6 +470,22 @@ The following table lists the measurement data that the HRM sensor provides.
 | Timestamp                   | `unsigned long long` | -                                   | Microseconds |
 | values[0]: Beats per minute | `int`                | Min. value = 0<br> Max. value = 240 | -            |
 
+<a name="hrm_batch"></a>
+## Heart Rate Monitor Batch Sensor
+
+The Heart Rate Monitor (HRM) Batch sensor measures a person's heart rate, it must be power-efficient and work in a batch manner.
+
+The following table lists the measurement data that the HRM sensor provides.
```

Comment:
According to the description, it would be "HRM batch sensor".
@bbrto21 , would you please check?

Also, I have a question. We can see the tables for HRM sensor and HRM batch sensor are different. Is there any reason for the different detected values between these two sensors?

Reply:
@mijin85cho They are different sensors and it is MCD requirements.